src/components/Table.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import {cn} from "../utils/util";
+
+
+export interface TableColumn {
+    field: string;
+    headerName: string;
+    width: number;
+    className?: string;
+}
+
+export interface TableRow {
+    [key: string]: React.ReactNode;
+}
+
+interface CardProps {
+    rows: TableRow[];
+    columns: TableColumn[];
+    className?: string;
+}
+
+export function Table({rows, columns, className}: CardProps) {
+    return (
+        <div className={cn("flex h-fit max-h-full w-full flex-col overflow-auto bg-gray-875", className)}>
+            <div className="flex h-fit w-max min-w-full flex-col">
+                <div className="sticky top-0 flex h-6 min-h-[24px] items-center gap-2.5 self-stretch overflow-x-hidden border-b border-b-fade-700 bg-gray-875 px-3">
```

Comment:
The header seems to have a scrollbar. Can we somehow remove that?
Also the min-height doesn't seem necessary?
![image](https://user-images.githubusercontent.com/44907675/225183325-b228dc38-d62d-4cd5-8bda-b1f458b8e39c.png)